doc/oil.txt

```diff
@@ -122,6 +122,10 @@ CONFIG                                                                *oil-confi
           return nil
         end,
       },
+      -- Set the default mode to create files
+      new_files_mode = 644,
+      -- Set the default mode to create folders
+      new_folder_mode = 755,
```

Comment:
I am fully aware of how this sounds, and I'm really sorry to be this pedantic, but "folder" is Windows terminology. Typically in Unix contexts we would refer to these as "directories". Elsewhere in this plugin we always refer to these as directories. For consistency sake, could we rename this to `new_dir_mode`?

Reply:
Done